arrays/java/UniqueDigitsOfLargeNumber.java

```diff
@@ -0,0 +1,62 @@
+import java.util.*;

```

Comment:
Rename file name from `UniqueDigitsOfLargeNumber` to `unique-digits-of-large-number`
> Only the file's name, not the class name
> Rember all file's name should be in lowercase and separate by `-`

Reply:
yeah done